### PR TITLE
Style place picker modal

### PR DESCRIPTION
**Pull request by jvaclavik**
**Web**

Before:
screenshot 2019-02-13 at 13 05 25

After:
screenshot 2019-02-13 at 13 04 16

**iOS**

Before:

After:

Btw why there is **Austria** when I search for **os**?

### CODE REVIEW

File: apps/core/scenes/search/PlacePicker/PlaceItem.js [hunk 1]
```diff
@@ -68,9 +78,16 @@ class PlaceItem extends React.Component<Props> {
...
-    alignItems: 'center',
+    alignItems: 'flex-end',
   },
-  text: {
+  iconContainer: {
+    paddingBottom: 8,
+  },
+  textContainer: {
+    flex: 1,
+    marginLeft: 5,
```
Comment (RobinCsl) [inline apps/core/scenes/search/PlacePicker/PlaceItem.js]: I thought units were meant to be divisible by 4?

File: apps/core/scenes/search/PlacePicker/PlacePickerList.js [hunk 2]
```diff
@@ -17,15 +18,23 @@ const PlacePickerList = (props: Props) => {
...
+        />
       ))}
     </ScrollView>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    marginStart: 25,
+    // marginStart: 25,
```
Comment (RobinCsl) [inline apps/core/scenes/search/PlacePicker/PlacePickerList.js]: Unnecessary comment.

File: apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js [hunk 3]
```diff
@@ -23,15 +25,24 @@ export default class PlacePickerRenderer extends React.Component<Props> {
...
-      <QueryRenderer
-        query={graphql`
-          query PlacePickerRendererQuery($input: LocationsByTermInput!) {
-            ...PlacePickerContent_locations @arguments(input: $input)
-          }
-        `}
-        variables={this.getInputVariable()}
-        render={this.renderInner}
-      />
+      <View style={styles.container}>
```
Comment (RobinCsl) [inline apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js]: I am not sold on moving the styles here. I think this component should only deal with rendering the query. Styling the container can be done in the Content component as it was before.

Comment (jvaclavik) [reply apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js]: But we are not able to style spinner inside of the Content component:

screenshot 2019-02-13 at 13 35 32

File: apps/core/scenes/search/PlacePicker/PlacePickerList.js (hunk 2 above)
Comment (jvaclavik) [reply apps/core/scenes/search/PlacePicker/PlacePickerList.js]: Deleted

File: apps/core/scenes/search/PlacePicker/PlaceItem.js (hunk 1 above)
Comment (jvaclavik) [reply apps/core/scenes/search/PlacePicker/PlaceItem.js]: True :) fixed

**Comment by jvaclavik** [comment]
Could anyone try it on Android? My Android simulator is not fetching data right now

**Comment by FilipMessa** [comment]
nice 🥇

File: apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js (hunk 3 above)
Comment (FilipMessa) [reply apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js]: I think we can improve our QueryRenderer to be able to render the custom component for loading. What do you think?
cc @tbergq

**Comment by FilipMessa** [comment]
> Could anyone try it on Android? My Android simulator is not fetching data right now

@jvaclavik  did you try this? `adb reverse tcp:4000 tcp:400`

File: apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js (hunk 3 above)
Comment (tbergq) [reply apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js]: I don't think our issue is with the loader but with our modal.

Comment (JoseDuSV) [reply apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js]: The picker still uses old modal implementation. After `universal-components` version update we will be able to replace it with new version.
So we can probably keep this style for now to make picker look better. And we will remove it after update.
Maybe just add some `todo` note to styles to keep this in mind.

Comment (jvaclavik) [reply apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js]: Sounds good, `@TODO` added

**Pull request merged**